test-resources/swift/cleanup_rules/expected/SampleClass.swift

```diff
@@ -251,4 +251,82 @@ class SampleClass {
             doSomething6b()
         }
     }
+
+    func refactorBooleanLiteralEqualityExpressionInIfConditions(){
+
+        toBePreservedEquality1()
+
+        toBePreservedEquality2()
+
+        toBePreservedEquality3() 
+
+        toBePreservedEquality4()
+
+        toBePreservedEquality5()
+
+        toBePreservedEquality6()
+
+        toBePreservedEquality7()
+
+        if a {
+        toBePreservedEquality8()
+        }
+
+        if b {
+        toBePreservedEquality9a()
+        } else {
+        toBePreservedEquality9b()
+        }
+
+        toBePreservedEquality10()
+
+        if c {
+            toBePreservedEquality11()
+        }
+
+        if d {
+            toBePreservedEquality12a()
+        } else {
+        toBePreservedEquality12b()
+        }
+    }
+
+    func refactorBooleanLiteralInequalityExpressionInIfConditions(){
+
+        toBePreservedInequality1()
```

Comment:
can we have a atleast 1/2 test cases testing the other literals (i.e. string, number, ...) ?